contracts/multisig/src/error.rs

```diff
@@ -25,8 +25,11 @@ pub enum ContractError {
     #[error("signed sender address could not be verified using submitted public key")]
     InvalidPublicKeyRegistrationSignature,
 
-    #[error("invalid public key format: {reason:?}")]
-    InvalidPublicKeyFormat { reason: String },
+    #[error("invalid public key")]
+    InvalidPublicKey,
+
+    #[error("ECDSA public key must be in compressed format (33 bytes)")]
+    InvalidEcdsaPublicKeyFormat,
```

Comment:
I feel it's a bit silly to have 2 errors here with one being the superset of the other.